_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,390 @@
+-->
+
+## Downloading and installing the Reporting CLI tool
+
+You can download and install the Reporting CLI tool from either the npm software registry or the OpenSearch.org [Download & Get Started](https://opensearch.org/downloads.html) page. Refer to the following sections for instructions.
+
+To learn more about the npm software registry, see the [npm](https://docs.npmjs.com/about-npm) documentation.
+
+### Downloading and installing the Reporting CLI from npm
+
+To download and install the Reporting CLI from npm, run the following command to initiate installation:
+
+```
+npm i @opensearch-project/opensearch-reporting-cli
+```
+
+### Downloading and installing the Reporting CLI from OpenSearch.org
+
+You can download the `opensearch-reporting-cli` tool from the OpenSearch.org [Command Line Tools](https://opensearch.org/downloads.html#command-line-tools) section.
+
+Next, run the following command to install the .tar archive:
+
+```
+npm install -g opensearch-reporting-cli-1.0.0.tgz
+```
+
+## Creating and requesting a visualization report
+
+First, you need to get the URL for the visualization that you want to download as an image file or PDF.
+
+To generate a visualization report, you need to specify the Dashboards URL.
+
+Open the visualization for which you want to generate a report, and select **Share >  Permalinks > Generate link as Shapshot > Short URL > Copy link**, as shown in the following image.
+
+![Copy link]({{site.url}}{{site.baseurl}}/images/dashboards/dash-url.png)
+
+You will need to add the URL with the `-u` argument when you request the report in the CLI.
+
+### Example: Requesting a PNG file
+
+The following command requests a report in PNG format with basic authentication and sends the report to an email address using Amazon SES:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:1234 -e ses -s <email address>  -r <email address> -f png
+```
+
+### Example: Requesting a PDF file
+
+The following command requests a PDF file and specifies the recipient's email address:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:Test@1234 -e ses -s <email address> -r <email address> -f pdf
+```
+
+Upon success, the file will be sent to the specified email address. The following image shows an example PDF report.
+
+![PDF example]({{site.url}}{{site.baseurl}}/images/dashboards/cli-pdf-report.png)
+
+### Example: Requesting a CSV file
+
+The following command generates a report that contains all table content in CSV format and sends the report to an email address using Amazon SES transport:
+
+```
+opensearch-reporting-cli -u https://search-basic-auth2-3-b477rrluhlckrmwmtvp6k3xxsi.us-west-2.es.amazonaws.com/_dashboards/goto/f166434919d2d8d3708a4705b6f2ed2d?security_tenant=global -f csv -a basic -c admin:Test1234 -e ses -s <email address> -r <email address>
```

Comment:
can we change this example to localhost/playground. This is my test domain info which might be deleted later.

Reply:
could you please share that link? thanks

Reply:
got it!